demos/stm32f10x/starter/project.mk

```diff
@@ -0,0 +1,87 @@
+# ======================
```

Comment:
Can you minimize this file to just 
```
PLATFORM = stm32f10x
```

Makes it easier to see what you have changed.